README.md

```diff
@@ -1,3 +1,18 @@
 # Literature Reviews
 
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review. You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request.
+
+## The process
+
+* Fork the https://github.com/elm-community/literature-reviews
+* Copy TEMPLATE.md to text/my-feature-name.md
+* Fill in the template
+* Submit a pull request. As a pull request the literature review will receive feedback from the larger community, and the author should be prepared to revise it in response.
+* Eventually, Evan will decide whether the feature is a candidate for inclusion in Elm.
+* If the feature is accepted the PR will be merged.
```

Comment:
I think we should go ahead an merge lit reviews that have approval from the community.